app/javascript/components/student/Editor.tsx

```diff
@@ -1,34 +1,102 @@
 import React, { useReducer, useRef, useEffect, useCallback } from 'react'
-import { CodeEditor } from './editor/CodeEditor'
 import { TestRunSummary } from './editor/TestRunSummary'
 import { Submitting } from './editor/Submitting'
 import { fetchJSON } from '../../utils/fetch-json'
 import { typecheck } from '../../utils/typecheck'
+import MonacoEditor from 'react-monaco-editor'
```

Comment:
I decided to put everything in one component for now since I was a bit unsure of how everything fits together. I hope this makes it easier for us to move things around, too.